client/my-sites/plugins/eligibility-warnings/index.jsx

```diff
@@ -0,0 +1,142 @@
+/**
+ * External dependencies
+ */
+import React, { Component, PropTypes } from 'react';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import Button from 'components/button';
+import Card from 'components/card';
+import Gridicon from 'components/gridicon';
+import SectionHeader from 'components/section-header';
+
+class EligibilityWarnings extends Component {
```

Comment:
Refactored as stateless functional component in 8b3ab4d82f850f5e3baa9ee0a73ef4995179f0d2.